loadouts/content/broma-weapon-list.sqf

```diff
@@ -356,28 +361,52 @@ _HLCMP5K = ["hlc_smg_mp5k_PDW",_30rnd9mmMP5];
 _ACRCOMPACTB = ["hlc_rifle_ACR_SBR_black",_30rnd556AR15];
 _ACRCOMPACTT = ["hlc_rifle_ACR_SBR_tan",_30rnd556AR15];
 _ACRCOMPACTG = ["hlc_rifle_ACR_SBR_green",_30rnd556AR15];
+_ACRCOMPACTBGL = ["hlc_rifle_ACR_GL_SBR_black",_30rnd556AR15,_40mm];
```

Comment:
`_ACRCOMPACTGLB` instead, same change to the other's. E.g. `_ACRCOMPACTTGL` to `_ACRCOMPACTGLT`, `_ACR68MIDTGL` to `_ACR68MIDGLT`, etc.